tests/extras/primitives/primitives/a-camera.test.js

```diff
@@ -1,14 +1,38 @@
-/* global assert, suite, test */
+/* global assert, process, setup, suite, test */
 suite('a-camera', function () {
-  test('is the active camera when applied', function (done) {
-    var scene = document.createElement('a-scene');
-    var camera = document.createElement('a-camera');
-    scene.appendChild(camera);
-    document.body.appendChild(scene);
+  'use strict';
 
+  setup(function (done) {
+    var sceneEl = this.sceneEl = document.createElement('a-scene');
+    var camera = this.camera = document.createElement('a-camera');
+    sceneEl.appendChild(camera);
+    document.body.appendChild(sceneEl);
+    if (camera.hasLoaded) { done(); }
     camera.addEventListener('loaded', function () {
+      done();
+    });
+  });
+
+  suite('active camera', function () {
+    test('is the active camera when applied', function (done) {
+      var camera = this.camera;
       assert.ok(camera.getComputedAttribute('camera').active);
       done();
     });
   });
+
+  suite('user-height', function () {
+    test('updates height offset when DOM user-height attibute changes', function () {
```

Comment:
If you have asynchrony on your test, you have to accept the `done` argument and call it when finished.

```js
function (done) {
  process.nextTick(function () {
    assert...
    camera.setAttribute...
    process.nextTick(function () {
      assert...
      done();
    });
  });
}
```